workshops/check.py

```diff
@@ -24,10 +10,14 @@
 import os
 import re
 import logging
-
 import yaml
 from collections import Counter
 
+try:  # Hack to make codebase compatible with python 2 and 3
+    basestring
+except NameError:
+    basestring = str
```

Comment:
The other alternative would be to use six

from six import string_types
isinstance(s, string_types)